### PR TITLE
DCOS_OSS-2355: show error if service was not found

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
To test this you can visit the edit page of a service and change the name in the URL you should see an error page

**Checklist**
- [X] Did you add a JIRA issue in a commit message or as part of the branch name?
- [X] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: plugins/services/src/js/components/modals/EditServiceModal.js [hunk 1]
```diff
@@ -6,17 +6,33 @@ import CreateServiceModal
...
+    const serviceLoaded = DCOSStore.serviceDataReceived;
     const service = DCOSStore.serviceTree.findItemById(serviceID);
 
-    if (!service) {
+    // Loading, showing an empty modal instead
+    if (!serviceLoaded) {
       return <FullScreenModal open={true} />;
     }
 
+    // Service not found
```
Comment (weblancaster) [inline plugins/services/src/js/components/modals/EditServiceModal.js]: I believe the ideal way is to add a route to the `routes/service.js` with a new not found path/route at the top level and not adding conditions to low level components.

```
{
    type: Route,
    path: "*",
    component: NotFoundPage
},
```

_ps: we actually have a `NotFoundPage` component._

**Comment by juliangieseke** [comment]
Ping @DanielMSchmidt

**Comment by DanielMSchmidt** [comment]
@juliangieseke Just rediscovered this 🙈 fix is pushed, please review :)

**Comment by juliangieseke** [comment]
@DanielMSchmidt do you think the thing mentioned by fabs is an easy win? if so we could try to implement it in this PR, otherwise I'd also suggest a follow up ticket 👍

**Comment by DanielMSchmidt** [comment]
@juliangieseke @Fabs I created an issue as there was no easy win here if I don't want to contradict @weblancaster request for an own route: https://jira.mesosphere.com/browse/DCOS_OSS-2370

**Comment by DanielMSchmidt** [comment]
@weblancaster Can you review this, please 🙏

File: plugins/services/src/js/components/modals/EditServiceModal.js (hunk 1 above)
Comment (nLight) [reply plugins/services/src/js/components/modals/EditServiceModal.js]: I don't think it's that easy. For that we would have to move the logic whether a service is there or not into the react router. While even though we don't have a service the route is still matchable.

**Comment by juliangieseke** [comment]
Ping @weblancaster

**Pull request merged**